src/Umbraco.Web.UI.Client/src/packages/tiptap/components/toolbar/tiptap-toolbar-menu.element.ts

```diff
@@ -7,10 +7,11 @@ import type { UmbCascadingMenuItem } from '../../components/cascading-menu-popov
 import { css, customElement, html, state, when } from '@umbraco-cms/backoffice/external/lit';
 import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';
 import type { Editor } from '@umbraco-cms/backoffice/external/tiptap';
+import type { ManifestMenu } from '@umbraco-cms/backoffice/menu';
 
 import '../../components/cascading-menu-popover/cascading-menu-popover.element.js';
 
-@customElement('umb-tiptap-toolbar-menu-element')
+@customElement('umb-tiptap-toolbar-menu')
```

Comment:
I'm aware this is considered a breaking-change, however this particular element should only be used from a Tiptap toolbar `menu` kind extension/manifest. It is doubtful that it would be used directly by a 3rd-party.

I'm happy to revert it and look to change for future version.